RUFAS/routines/manure/manure_handlers/manure_handler_classes.py

```diff
@@ -222,6 +223,17 @@ class Tillage(BaseManureHandler):
     pass
 
 
+class Harrowing(BaseManureHandler):
+    """A class that handles calculations related to harrowing.
+
+    Attributes:
+        All inherited from BaseManureHandler.
```

Comment:
```suggestion
    Attributes
    ----------
    All inherited from BaseManureHandler.
```
Is there a plan to add harrowing-specific attributes at some point?

Reply:
Currently, all the manure handler classes don't seem to have specific attributes associated with them. They inherit from the same base manure handler class. So, I'm thinking during a future refactoring, these separate manure handler subclasses could be consolidated and removed.